tests/README.md

```diff
@@ -7,6 +7,11 @@ The backend test suite can be run outside the Docker container.
 We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
 
     # ENSURE THE IMAGES ARE CLEAN
+    docker rm "/mysql"
+    docker rm "/frontend"
+    docker rm "/rabbitmq" 
+    docker rm "/redis"
+    docker rm "/backend"
```

Comment:
When do you need to do this?
I've always found it enough with `docker-compose down`.

Reply:
this was needed today when I cloned treeherder to make the isort example.  docker-compose down was not enough:

```
C:\Users\kyle\code\treeherder2>    docker-compose down
Removing network treeherder2_default

C:\Users\kyle\code\treeherder2>    docker volume rm treeherder_mysql_data
treeherder_mysql_data

C:\Users\kyle\code\treeherder2>docker-compose up --build
Creating network "treeherder2_default" with the default driver
Building backend
Step 1/15 : FROM python:3.7.2
 ---> 2053ca75899e
Step 2/15 : ENV NEW_RELIC_CONFIG_FILE newrelic.ini
 ---> Using cache
 ---> aa16c195b5b0
Step 3/15 : RUN apt-get update && apt-get install -y --no-install-recommends     default-libmysqlclient-dev     gcc     netcat     && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 104d64da1ca9
Step 4/15 : ENV GECKODRIVER_VERSION='0.24.0'
 ---> Using cache
 ---> 3643d63382d3
Step 5/15 : RUN curl -sSfL --retry 5 "https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz"     | tar -zxC "/usr/local/bin"     && curl -sSfL --retry 5 'https://download.mozilla.org/?product=firefox-beta-latest&lang=en-US&os=linux64'     | tar -jxC "/usr/local/bin"
 ---> Using cache
 ---> b81bdbf1b34a
Step 6/15 : RUN apt-get update && apt-get install -y --no-install-recommends     libgtk-3-0     libdbus-glib-1-2     && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> fb2ea11814bb
Step 7/15 : ENV MOZ_HEADLESS=1
 ---> Using cache
 ---> 6fac16c6088d
Step 8/15 : ENV PATH="/usr/local/bin/firefox:${PATH}"
 ---> Using cache
 ---> f4e18ff2ebb1
Step 9/15 : ENV SHELLCHECK_VERSION="0.4.6"
 ---> Using cache
 ---> 01fa9b02ee8a
Step 10/15 : RUN curl -sSfL --retry 5 "https://storage.googleapis.com/shellcheck/shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz"     | tar -Jx --strip-components=1 -C /usr/local/bin
 ---> Using cache
 ---> d1d879b24cd1
Step 11/15 : ADD . /app
 ---> Using cache
 ---> 60a804c71345
Step 12/15 : WORKDIR /app
 ---> Using cache
 ---> 566df236c403
Step 13/15 : RUN pip install --no-cache-dir --disable-pip-version-check --require-hashes -r requirements/common.txt
 ---> Using cache
 ---> c3328f6e55a3
Step 14/15 : RUN pip install --no-cache-dir --disable-pip-version-check --require-hashes -r requirements/dev.txt
 ---> Using cache
 ---> ae55c4a33058
Step 15/15 : RUN pip install --no-cache-dir --disable-pip-version-check -r requirements/docs.txt
 ---> Using cache
 ---> 4d187dbfa63f
Successfully built 4d187dbfa63f
Successfully tagged treeherder-backend:latest
Creating frontend ... done
Creating mysql    ... done
Creating rabbitmq ... done
Creating redis    ... done
Creating backend  ... error

ERROR: for backend  Cannot create container for service backend: Conflict. The container name "/backend" is already in use by container "df4a65f70f9fc0a1a0ff17badf0d2702be2c8af33027a1904968556397e4b154". You have to remove (or rename) that container to be able to reuse that name.

ERROR: for backend  Cannot create container for service backend: Conflict. The container name "/backend" is already in use by container "df4a65f70f9fc0a1a0ff17badf0d2702be2c8af33027a1904968556397e4b154". You have to remove (or rename) that container to be able to reuse that name.
ERROR: Encountered errors while bringing up the project.

C:\Users\kyle\code\treeherder2>
C:\Users\kyle\code\treeherder2>docker rm "/mysql"
Error response from daemon: You cannot remove a running container 8cbc51c7e51387c35a1037955f32a88fb39828abc3ad5afcb4d7bbd710fe8e2a. Stop the container before attempting removal or force remove

C:\Users\kyle\code\treeherder2>docker rm "/frontend"
Error response from daemon: You cannot remove a running container f5d3b68afb24c0b4c8e89a9e445ae4d4d8be0249bf1a5da936b6a0776252da59. Stop the container before attempting removal or force remove

C:\Users\kyle\code\treeherder2>docker rm "/rabbitmq"
Error response from daemon: You cannot remove a running container 1dd26300e86ed4fdc9c9e274e43bb151d08f1356ad921d6d9d1db87362d401ec. Stop the container before attempting removal or force remove

C:\Users\kyle\code\treeherder2>docker rm "/redis"
Error response from daemon: You cannot remove a running container 3e02f74f38cae95cd8e191a2ab72165daa85f6d1c1aac66f4d32e19351aa4f57. Stop the container before attempting removal or force remove

C:\Users\kyle\code\treeherder2>docker rm "/backend"
/backend
```

Reply:
That's really bizarre! I've not seen it before.